src/base/playback/playback.js

```diff
@@ -23,10 +23,10 @@ import $ from 'clappr-zepto'
  */
 export default class Playback extends UIObject {
   /**
-  * Determine if the playback does not contain video/has video but video should be ignored.
-  * @property isAudioOnly
-  * @type Boolean
-  */
+   * Determine if the playback does not contain video/has video but video should be ignored.
+   * @property isAudioOnly
+   * @type Boolean
+   */
```

Comment:
remove this white space

Reply:
Done in [Commit](https://github.com/clappr/clappr-core/pull/120/commits/4182cd4e5e8bcec65dfbd25174874336999cd637)